website/content/gateway/api_reference/retry_message/retry_message.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Retry message"
+anchor: "retry-message"
+weight: 207
+---
+
+## Retry message
+
+If an `authorization` has been rejected a retry message _can_ be sent in the
+response in addition to the `status`. The message will tell if the transaction
+can be retried now, later or never.
+
+```json
+{
+	"retry": {
+		"with": [
```

Comment:
Or "criteria" or "conditions". I'm more for semantic naming than "natural reading".